docs/guides/advanced/schemas.md

```diff
@@ -547,6 +547,30 @@ for (let customField of customFeilds) {
 ```
 
 
+### Column names {: #column-names}
+
+By default sync table column names are derived from the [normalized name](#normalization) of the corresponding schema property. For example, a property with the key `contact_email` would be normalized to the property name `contactEmail` which would result in a column named "Contact email". This is just the default name for the column however, and users can later rename it in their doc if they so desire.
```

Comment:
technically i think the normalized form is `ContactEmail`

Reply:
Whoops, good catch. Will send a follow-up PR.